blocks/tabs/tabs.css

```
@@ -1,58 +1,174 @@
 .tabs .tabs-list {
-    display: flex;
-    gap: 0.5ch;
+  display: flex;
+  gap: 0.5ch;
```

```suggestion
  gap: 12px;
```
Let's just make the gap 12px since that's what the 2 IDFC tabs are using.